### DISCUSSION TITLE
Formula Preview

### DISCUSSION THREAD

**Discussion by gyl0k** [texstudio-org/texstudio#3621] [Q&A] [answered]
I use the TeXstudio + Xelatex configuration to set my documents. 

I have a problem previewing formulas. This function just doesn't work. I have tried using various preview functions (dvipng, dvips, pdflatex, lualatex), but they give an error. The paths for dvipng, dvips, pdflatex, lualatex are configured correctly. Tell me how this can be fixed (using Xelatex specifically). 

Here is an example of a minimal document that is compiled, but in which formula preview does not work:
\documentclass[a4paper,14pt]{extarticle}

\usepackage{polyglossia}
\setdefaultlanguage{russian}
\setotherlanguage{english}
\setmainfont{Times New Roman}

\begin{document}
	Предпросмотр
	$ y = ax^2 + bx + c + t $
\end{document}

**Comment by octaeder** [comment, marked as answer]
Should be resolved by #3623 with next release.
For a preview/test you may download now a current build artifact from this page: [preview with xelatex (#3623)](https://github.com/texstudio-org/texstudio/actions/runs/8868500405)
Select _Preview with xelatex_ from the preview options.

**Reply by gyl0k** [reply]
Thanks, it works!

**Comment by octaeder** [comment]
Hints from tex.stackexchange may be helpful [https://tex.stackexchange.com/questions/49295/precompile-header-with-xelatex](https://tex.stackexchange.com/questions/49295/precompile-header-with-xelatex)
